tmt/base.py

```diff
@@ -86,6 +86,12 @@ def __init__(self, node, parent=None):
         # Check whether the node is enabled, handle the default
         self._check('enabled', expected=bool, default=True)
 
+        # Check whether the order is integer, handle the default
+        self._check('order', expected=int, default=0)
```

Comment:
Let's create a new constant for the default order and use it here.

```suggestion
        self._check('order', expected=int, default=DEFAULT_ORDER)
```

Reply:
Outcome from today's hacking discussion: The default order should be `50` to make it consistent with what we already have defined for multiple `prepare` step configs.